src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -51,7 +52,18 @@ class MockWorkflowManagerActor extends Actor  {
 
   def receive = {
     case SubmitWorkflow(sources) =>
-      sender ! MockWorkflowManagerActor.submittedWorkflowId
+      val id = MockWorkflowManagerActor.submittedWorkflowId
+      /*
```

Comment:
@mcovarr I resurrected this w/ comment - it's not a simple fromTry. Do you see a better path here?